data/json/professions.json

```diff
@@ -6265,5 +6305,605 @@
+    ],
+    "proficiencies": [ "prof_gunsmithing_basic", "prof_spotting", "prof_wp_syn_armored", "prof_gun_cleaning" ],
+    "items": {
+      "both": {
+        "items": [
+          "fancy_sunglasses",
+          "platinum_watch",
+          "polaroid_photo",
+          "dress_shirt",
+          "dress_shoes",
+          "socks",
+          "hat_cotton",
+          "tie_skinny"
+        ],
+        "entries": [
+          { "group": "charged_cell_phone" },
+          { "item": "pants", "variant": "pants_black", "contents-group": "ccp_mags" },
+          {
+            "item": "weatherby_5",
+            "ammo-item": "300_winmag",
+            "charges": 3,
+            "contents-item": [ "shoulder_strap", "suppressor", "bipod", "rifle_scope" ]
+          },
+          { "item": "300_winmag", "charges": 20 },
+          { "item": "walther_ccp", "ammo-item": "9mmfmj", "charges": 8, "container-item": "holster" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    },
+    "missions": [ "MISSION_ASSASSINATION" ]
+  },
+  {
+    "type": "profession",
+    "id": "carpenter",
+    "name": "Carpenter",
+    "description": "You spent most of your days in the workshop making furniture for a living.  Let's hope whatever you build is strong enough to keep the undead at bay.",
+    "points": 2,
+    "skills": [
+      { "level": 6, "name": "fabrication" },
+      { "level": 3, "name": "melee" },
+      { "level": 3, "name": "bashing" },
+      { "level": 2, "name": "swimming" }
+    ],
+    "proficiencies": [ "prof_carving", "prof_carpentry_basic" ],
+    "items": {
+      "both": {
+        "items": [
+          "tshirt",
+          "gloves_work",
+          "pants_cargo",
+          "socks",
+          "boots",
+          "leather_belt",
+          "wristwatch",
+          "mask_dust",
+          "hammer",
+          "glasses_safety"
+        ],
+        "entries": [
+          { "group": "charged_smart_phone" },
+          { "item": "saw", "custom-flags": [ "auto_wield" ] },
+          { "item": "nail", "charges": 50 }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "food_critic",
+    "name": "Food Critic",
+    "description": "Part of your job description included tearing restaurants to shreds on your online blog.  Too bad the zombies aren't much for fine dining.",
+    "points": 2,
+    "skills": [ { "level": 6, "name": "speech" }, { "level": 3, "name": "cooking" }, { "level": 2, "name": "computer" } ],
+    "traits": [ "PICKYEATER" ],
+    "items": {
+      "both": {
+        "items": [ "suit", "dress_shoes", "socks", "mbag" ],
+        "entries": [ { "group": "charged_smart_phone" }, { "group": "charged_laptop" } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "bootlegger",
+    "name": "Bootlegger",
```

Comment:
While people do probably bootleg liquor professionally, it seems much more like a hobby.